pkgs/by-name/xf/xfig/package.nix

```diff
@@ -39,8 +39,8 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
-    substituteInPlace src/main.c --replace '"fig2dev"' '"${fig2dev}/bin/fig2dev"'
-    substituteInPlace xfig.desktop --replace "/usr/bin/" "$out/bin/"
+    substituteInPlace src/main.c --replace-fail '"fig2dev"' '"${fig2dev}/bin/fig2dev"'
+    substituteInPlace org.xfig.xfig.desktop --replace-fail "Exec=xfig" "Exec=$out/bin/xfig"
```

Comment:
.desktop files should not hard link a binary, see #308324 for information on why.

```suggestion
```